standard/ERCs/erc-6900.md

```diff
@@ -44,8 +44,8 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
   - **Validation functions** validate authorization on behalf of the account.
   - **Execution functions** execute custom logic allowed by the account.
   - **Hooks** execute custom logic and checks before and/or after an execution function or validation function. There are two types of hooks:
-    - **Validation hook** functions run before a validation function. These can enforce permissions on actions authorized by a validation function.
-    - **Execution hook** functions can run before and/or after an execution function. A pre execution hook may optionally return data to be consumed by a post execution hook function.
+    - **Validation hook** functions run before a validation function. These can enforce permissions on actions authorized by a validation function and are limited by restrictions from ERC4337 in the validation phase.
```

Comment:
Do we want to mention this here? Technically the 4337 restrictions were spun out to a different ERC, ERC-7562.

Also nit: if we do want to keep it, we need to change it to "ERC-4337"

Reply:
I added it to differentiate why we need pre val hooks + exec hooks assoc with a validator, but fine leaving it out also